hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/LockCodecTemplate.java

```diff
@@ -68,27 +68,30 @@
      * @param name      Name of the Lock
      * @param leaseTime Time to wait before releasing to lock
      * @param threadId  The id of the user thread performing the operation. It is used to guarantee that only the lock holder thread (if a lock exists on the entry) can perform the requested operation.
+     * @param referenceId The client-wide unique id for this request. It is used to make the request idempotent during retries.
```

Comment:
Ok, changed the explanation like this:
"It is used to make the request idempotent by sending the same reference id during retries."